index.src.html

```diff
@@ -519,33 +554,34 @@ <h3 id="integration-websockets">Integration with WebSockets</h3>
 
   <h3 id="integration-html">Integration with HTML</h3>
 
-  To support the checks in [[FETCH]], we store an <dfn>address space</dfn>
-  on both {{Document}} and {{WorkerGlobalScope}} objects, which is set as
-  follows during {{Document}} and {{Worker}} initialization:
+  To support the checks in [[FETCH]], we store an
+  <dfn>address space property</dfn> on both {{Document}} and
```

Comment:
I'd suggest that you move "property" out of the definition, and tie it to the object via `<dfn for="Document">`.

Reply:
I guess what I'm really asking for is for this section to be more monkey-patchey, similar to what we did in https://wicg.github.io/cross-origin-embedder-policy/#integration-html (define the terms for each type, and point to the places where the property is set. That's probably a bit more work than this patch needs to do...

Reply:
Fair enough. I'll take a look at what COEP does and copy it in a followup patch. In the meantime, I removed `property` from the definition itself.

Had to make the dfn an `attribute` otherwise it collided with the regular `address space` definition. Contrary to what I can find in Bikeshed's documentation:

> To link specifically to the version of a definition without a for value, use an empty slash, like [=/url=]

I was not able to pacify it by using `[=/address space=]`.

Reply:
Perhaps you could just define the `<dfn for="Document">address space</dfn>` variant (referenced via `[=Document/address space=]`) and use that until we do the work of further integrating into HTML?